packages/react-kit/src/components/EventListener/EventListener.tsx

```diff
@@ -51,7 +51,8 @@ export class EventListener extends React.Component<TEventListenerProps> {
 			const capture = key.endsWith(CAPTURE_MARKER);
 			const handler = handlers[key];
 			const eventName = getEventName(key, capture);
-			target.addEventListener(eventName, handler as any, capture);
+			const passive = !['touchcancel', 'touchend', 'touchmove', 'touchstart'].includes(eventName);
```

Comment:
are you sure you want to process _all_ events in passive mode?

Reply:
I [update](https://github.com/devexperts/dx-platform/pull/212/commits/6db8313000f284d4aef0c7b1fdef4127a722092a) it